tests/conftest.py

```diff
@@ -246,3 +248,24 @@ def telescope_north_test_file():
 @pytest.fixture
 def telescope_south_test_file():
     return "data/layout/telescope_positions-South-TestLayout.ecsv"
+
+
+@pytest.fixture
+def corsika_output_file_name_string():
```

Comment:
I don't think you mean `string` here right? Is it actually a prefix? 
Why isn't there a suffix to this file name? (The answer to that might be made clear below, so I hope I remember to remove this question if that's the case.)

Reply:
I dont understand the question. 'string' is a suffix here (it comes after the function name). and the suffix indicates that the file returned by this function is a string and not yet Path.

Reply:
If you added `string` just to note that it's not a `Path`, then I think it's not necessary and you can remove it. The point about the prefix is related to my question about the suffix. I.e., the file name doesn't end with `.txt` or `.dat` or whatever. I was wondering why and if it is intentional. Perhaps this name is actually a file "`stem`" (see the pathlib `stem` function) to which you add different suffixes in various tests. In that case I would replace `string` with `stem`.

Reply:
Actually that is intentional because the test file has no extension. The output is in binary. Do you suggest adding an extension to the file name?

I cannot simply remove "_string" because of the fixture just below it, which would have the same name. Both fixtures are used in different places and are necessary.